python/src/EnumerateFunctionImplementation_doc.i.in

```diff
@@ -133,12 +133,12 @@ max_deg : int
 Returns
 -------
 index : int
-    Index of the last strata that contains terms of total degree :math:`\leq max_deg`.
+    Index of the last strata that contains multi-indices of total degree :math:`\leq \max_{deg}`.
 
 Notes
 -----
 In the specific context of a linear enumeration (:class:`LinearEnumerateFunction`)
-this is the strata of index *max_deg*.
+this is the strata of index `max_deg`.
```

Comment:
```suggestion
this is the strata of index *max_deg*.
```
I propose to revert this change. Traditionally, we use italics to refer to parameter names. If we want to change this, we need to decide it during a technical committee meeting.